docs/source/spec/aws-json-protocols.rst

```diff
@@ -0,0 +1,103 @@
+.. _aws-json-protocols:
+
+==================
+AWS JSON protocols
+==================
+
+This specification defines the ``aws.protocols#awsJson1_0`` and
```

Comment:
Even though the namespace has `aws` in it, I think it's a good idea to keep `aws` in the trait name so it's more easily discerned from other JSON traits when using `use` statements.